File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {
    List<TbBrand> findAll();

    PageResult findPage(TbBrand brand,int pageNum, int pageSize);

    void save(TbBrand brand);

    void update(TbBrand brand);

    TbBrand findOne(Long id);

    void delete(Long id);

    List<Map> selectOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findAll = function () {
        return $http.get("../brand/findAll.do");
    }
    this.search = function (searchEntity,pageNum,pageSize) {
        return $http.post("../brand/findPage.do?pageNum="+pageNum+"&pageSize="+pageSize,searchEntity);
    }
    this.findOne =function (id) {
        return $http.get("../brand/findOne.do?id="+id);
    }
    this.add = function (entity) {
        return $http.post("../brand/save.do",entity)
    }
    this.update = function (entity) {
        return $http.post("../brand/update.do",entity)
    }
    this.delete = function (ids) {
        return $http.get("../brand/delete.do?ids="+ids)
    }

    $scope.selectOptionList = function () {
        return $http.get("../brand/selectOptionList.do")
    }
})